MathProblems/326_Power_of_Three.py

```
@@ -0,0 +1,18 @@
+        while x<n:
+             x*=3 
+             if x==n: return True 
+        return False
+import math
+class Solution:
+    def isPowerOfThree(self, n: int) -> bool:
+        if n<=0: return False 
+        return  math.floor(math.log(n)/math.log(3)) == math.ceil(math.log(n)/math.log(3))
+class TestApp:
+        def testing_case_one(self):
+            assert Solution().isPowerOfThree(27)==True 
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if Solution().isPowerOfThree(27):` for truth checks

Replace with `Solution().isPowerOfThree(27)`

(E712)